kmk/modules/layers.py

```diff
@@ -47,8 +38,8 @@ def __init__(self):
         make_argumented_key(
             validator=layer_key_validator,
             names=('LT',),
-            on_press=self._lt_pressed,
-            on_release=self._lt_released,
+            on_press=curry(self.ht_pressed, key_type='LT'),
```

Comment:
Preference to restoring the const values or at least storing these in variables somewhere: magic strings are prone to mistakes.

The const values that previously existed are a fair bit RAM-lighter than these strings will be, which doesn't so much matter on some of the beefier NRF and RP2040 boards of today, but does still matter on older/smaller boards running on M4 or STM32.